spl/src/metadata.rs

```diff
@@ -183,3 +183,31 @@ pub struct MintNewEditionFromMasterEditionViaToken<'info> {
     //
     pub metadata_mint: AccountInfo<'info>,
 }
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct MetadataAccount(mpl_token_metadata::state::Metadata);
```

Comment:
I think it will collide with the other Metadata account that's for the Metadata program, there's a similar occurrence in the token account one, can do, just trying to adapt to the repo